src/App.vue

```diff
@@ -783,7 +784,11 @@ onMounted(() => {
   if ("midiOctaveOffset" in storage) {
     midiOctaveOffset.value = parseInt(storage.getItem("midiOctaveOffset")!);
   }
-
+  if ("intervalMatrixIndexing" in storage) {
+    intervalMatrixIndexing.value = parseInt(
+      storage.getItem("intervalMatrixIndexing")!
```

Comment:
Still doing naughty things and storing a `NaN` on first load.

Reply:
This really should be `parseInt(storage.getItem("intervalMatrixIndexing") ?? "0", 10)`